### PR TITLE
Refine instance host tables

### PR DESCRIPTION
**Pull request by baurine**
What did:

- Refine InstanceTable, add tooltip for version/deploy_path/git_hash columns, add dummyColumn
- Refine HostTable, add tooltip for disk column

Before:

企业微信20200511064550

After:

企业微信20200511062810

### CODE REVIEW

File: ui/lib/apps/ClusterInfo/components/InstanceTable.tsx [hunk 1]
```diff
@@ -153,21 +154,18 @@ export default function ListPage() {
...
-      onRender: (node) => (
-        <Tooltip title={`${node.ip}:${node.port}`}>
-          <span>
-            {node.ip}:{node.port}
-          </span>
-        </Tooltip>
-      ),
+      onRender: ({ ip, port }) => {
+        const fullName = `${ip}:${port}`
+        return <Tooltip title={fullName}>{fullName as any}</Tooltip>
```
Comment (breeswish) [inline ui/lib/apps/ClusterInfo/components/InstanceTable.tsx:162]: `<Tooltip>` needs some element node I believe, otherwise it won't take effect.

Comment (baurine) [reply ui/lib/apps/ClusterInfo/components/InstanceTable.tsx:162]: But actually it works, the tooltip can be displayed normally, just has type warning.

I just have a look at the Tooltip source code in https://github.com/ant-design/ant-design/blob/master/components/tooltip/index.tsx#L245 :

```js
    const child = getDisabledCompatibleChildren(
      React.isValidElement(children) ? children : <span>{children}</span>,
      prefixCls,
    );
```

So the Tooltip will wrap the string with `<span>`.

Anyway, it is better to pass the element node as children, I will update it soon.

Comment (breeswish) [reply ui/lib/apps/ClusterInfo/components/InstanceTable.tsx]: Good to know! I didn't know it before.

**Comment by breeswish** [comment]
I have an idea. We can always append the dummy column in CardTableV2. (we can do it in another PR)

**Comment by baurine** [comment]
> I have an idea. We can always append the dummy column in CardTableV2. (we can do it in another PR)

Right, brilliant!

**Pull request merged**